spec/lita/handler/event_router_spec.rb

```diff
@@ -61,5 +71,41 @@ def greet(payload)
       subject.trigger(robot, "connected")
       subject.trigger(robot, " ConNected  ")
     end
+
+    it "doesn't stop triggering callbacks after an exception is raised" do
+      test_mode = Lita.test_mode?
+      begin
```

Comment:
Use an `around` block in a nested describe block to handle the global test_mode state for these three tests:

``` ruby
describe "error handling" do
  around do |example|
    test_mode = Lita.test_mode?
    example.run
    Lita.test_mode = test_mode
  do

  it "doesn't stop trigger callbacks after an exception is raised" do
    # ...
  end
end
```